docs/sources/reference/commandline/cli.md

```diff
@@ -1310,6 +1310,22 @@ timestamp, for example `2014-09-16T06:17:46.000000000Z`, to each
 log entry. To ensure that the timestamps for are aligned the
 nano-second part of the timestamp will be padded with zero when necessary.
 
+## pause
+
+    Usage: docker pause CONTAINER
+
+    Pause all processes within a container
+
+The `docker pause` command uses the cgroups freezer to suspend all processes in
```

Comment:
As this is moving existing content I think we can look into changing wording later in another PR if you think function or another term is better